core/new-gui/src/app/workspace/component/left-panel/operator-menu/operator-label/operator-label.component.scss

```diff
@@ -0,0 +1,57 @@
+.disable-drag-drop {
+  color: rgb(151, 151, 151);
+  &:hover {
+    border-color: rgb(217, 217, 217);
+  }
+}
+
+.operator-label {
+  width: 100%;
+  height: calc(100% - 1px);
+  display: flex;
+  align-content: center;
+  border-radius: 3px;
+  border: 1px solid rgba(0, 0, 0, 0);
+  transition: border-color 0.3s, color 0.3s;
+  overflow: hidden;
+  &:hover {
+    border-color: rgba(64, 169, 255, 1);
+    > .text {
+      transform: translateX(10px);
+    }
+  }
+  > .text {
+    user-select: none;
+    transition: transform 0.5s 0.1s;
+  }
+}
+
+.shake-animation {
+  animation: shake 0.5s linear;
+}
+
+@keyframes shake {
```

Comment:
is @keyframe a standard css notion? my understanding is limited. just want to make sure it is compatible in all modern browsers (i.e., including safari etc).

Reply:
It's standard, and it's from the original file, i.e., it's not a new CSS introduced by this PR.